deployment/helm/skaha/sample-local-values.yaml

```diff
@@ -35,6 +32,18 @@ deployment:
       minEphemeralStorage: "10Gi" # Initial request for ephemeral storage (scratch disk space)
       maxEphemeralStorage: "100Gi" # Max expansion for ephemeral storage (scratch disk space)
 
+      # When set to 'true' this flag will enable GPU node scheduling.
+      gpuEnabled: true
+
+      # Example nodeAffinity to restrict how User Sessions are scheduled.
+      nodeAffinity:
+        # Only allow executions of User Sessions on specific Nodes.
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: high-compute-batch-nodes
```

Comment:
The example here could be our system nodes (persistent nodes) to show our use case.